### PR TITLE
Feature/delete comment

### PR DESCRIPTION
**Pull request by davidporosnicu**

### CODE REVIEW

**Review by horiaradu — CHANGES_REQUESTED** [review]
This can't be reviewed because it has changes from other PRs. We need to merge #9 first

File: src/components/BlogForm.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import React, { Component } from 'react';
 import { addNewEntry, updateEntry } from '../redux/actions/blogActions';
 import { connect } from 'react-redux';
+import { Redirect } from 'react-router-dom';
```
Comment (horiaradu) [inline src/components/BlogForm.js]: Why?

File: src/redux/actions/blogActions.js [hunk 2]
```diff
@@ -39,3 +39,13 @@ export const updateEntry = (entryUuidToUpdate, updatedEntry, userId) => dispatch
...
   dispatch({ type: UPDATE_ENTRY, entryUuidToUpdate, updatedEntry });
 };
+
+export const deleteComment = (commentUuid, entryUuid2) => async dispatch => {
+  const listOfFilteredComments = await api.deleteComment(commentUuid, entryUuid2);
+
+  dispatch({
+    type: 'DELETE_COMMENT',
+    listOfFilteredComments,
+    entryUuid2
```
Comment (horiaradu) [inline src/redux/actions/blogActions.js]: `entryUuid2`?

File: src/redux/reducers/blogReducers.js [hunk 3]
```diff
@@ -71,6 +72,21 @@ export default function(state = initialState, action) {
         ...state,
         entries: newState
       };
+    case 'DELETE_COMMENT':
+      const { entryUuid2, listOfFilteredComments } = action;
```
Comment (horiaradu) [inline src/redux/reducers/blogReducers.js]: `entryUuid2`?

**Review by horiaradu — CHANGES_REQUESTED** [review]

File: src/components/BlogForm.js (hunk 1 above)
Comment (davidporosnicu) [reply src/components/BlogForm.js]: i forgot to delete it after i used hoc for route protection. it's all good now

**Pull request merged**